torch/nn/modules/activation.py

```diff
@@ -964,6 +964,8 @@ class MultiheadAttention(Module):
 
     def __init__(self, embed_dim, num_heads, dropout=0., bias=True, add_bias_kv=False, add_zero_attn=False,
                  kdim=None, vdim=None, batch_first=False, device=None, dtype=None) -> None:
+        if embed_dim <= 0 or num_heads <= 0:
+            raise ValueError("embed_dim and num_heads must be greater than 0, got embed_dim={embed_dim} and num_heads={num_heads} instead")
```

Comment:
This should be an f-string
```suggestion
            raise ValueError(f"embed_dim and num_heads must be greater than 0, got embed_dim={embed_dim} and num_heads={num_heads} instead")
```